lib/compiler-singlepass/src/codegen_x64.rs

```diff
@@ -6334,14 +6337,14 @@ impl<'a> FuncGen<'a> {
 
                 self.assembler.emit_label(after);
             }
-            Operator::BrTable { ref table } => {
-                let mut targets = table
+            Operator::BrTable { ref targets } => {
+                let default_target = targets.default();
```

Comment:
Uh-oh, that’s a footgun. I think I probably made a mistake of not considering the default target in my gas/max_stack work T_T.

Reply:
Looks to me like you’re find and did think about it for gas, and stack doesn’t have anything to do with it :)